azure-pipelines.yml

```diff
@@ -201,6 +201,17 @@ jobs:
       buildType: 'current'
       downloadType: 'specific'
       downloadPath: '$(Build.SourcesDirectory)'
```

Comment:
One suggestion, we can remove the Build.SourcesDirectory and change it to ArtifactStagingDirectory  from the download path in line 203. In this way, your sbom generation task will use Build.ArtifactStagingDirectory  and you can remove the CopyFiles task

Reply:
good idea .. I will experiment with it .. I believe I tried it and it didn't work for some reason .. but I will try again. I merged my PR .. but I can send a new one to reflect that change